### ISSUE TITLE
Starting BME280... result of .begin(): 0x0 not shows values properly

### ISSUE THREAD

**Issue by gregbartels** [IU5HKU/MiniWXStation#10]
Hello,
I'm experiencing some problems and I'm very frustrated at this point.

When I tried to test the sensor I got this:

Program Started
Starting BME280... result of .begin(): 0x0
Pressure at home level: 0.00 Pa
Pressure at sea level: 0.00 Pa
Temperature: 0.00 degrees C
Temperature: 32.00 degrees F
%RH: 0.00 %

![image](https://user-images.githubusercontent.com/78452640/125138025-19288180-e0cb-11eb-8b58-4bb95ceadb5a.png)

However, the sensor is connected as required and accordingly the diagram and wire information shared
![image](https://user-images.githubusercontent.com/78452640/125138753-77a22f80-e0cc-11eb-8d1b-fd8cb62a1d49.png)
.

I would like to know if there is something I Can check to make this work station work, this is my  sixty BME280 and ESP8266 and I'm still not having results to configure properly the sensor. I will appreciate any feedback.

 I'm using a HitLetgo ESP8266 and hitletgoBME8280 , Below you will see the screenshot with the behavior I'm experiencing.

Thanks for the help!
TI2GBB - Greg

**Comment by df2ap** [comment]
I have the same Problem. There is a Problem with newer Versions of the Sparkfun Library for the BME280. Go back to Version 2.08 an I think it will work.

**Comment by gregbartels** [comment]
> I have the same Problem. There is a Problem with newer Versions of the Sparkfun Library for the BME280. Go back to Version 2.08 an I think it will work.

YOU'RE A GENIUS!!!!!! That fixes all the problems!!!! Now I can see data coming through the sensor! MANY MANY MANY MANY THANKS!!
![image](https://user-images.githubusercontent.com/78452640/125155300-12762a80-e11c-11eb-99ea-7754ef105510.png)


Thank you so much for the response.  I checked that on my arduino, I hope this fix the issue for others  this project is Awesome and I'm trying to install one of this weather stations in my country Costa Rica, its sad that the developer haven't upgrade the content of the code to the new libraries to avoid the issues listed previously, but fortunately we have the workaround with the library you mentioned.

In case you experienced an issue like this one, just proceed to install the library https://downloads.arduino.cc/libraries/github.com/sparkfun/SparkFun_BME280-2.0.8.zip

This will resolve the issues with the sensor.


Do you know if this is the expected behavior when you tried to send packages to APRS? I'm using my callsing, I have around 40 minutes trying to see my data in APRS without not luck ( I'm using TI2GBB-13) 
![image](https://user-images.githubusercontent.com/78452640/125155333-4b160400-e11c-11eb-89c3-0c19c9333f9a.png)


I will appreciate any advice!  73s from Costa Rica
TI2GBB - Greg

**Comment by gregbartels** [comment]
I fixed the problem with the data being sent to aprs, I use APRSDROID to correct my location. 
![image](https://user-images.githubusercontent.com/78452640/127938727-143cca9b-5a59-4edb-90b8-56c0477a9474.png)


thanks for the help!

**Comment by VK6GHZ** [comment]
Hi All,

Could someone please advise if this issue has been fixed in V1.1 of the code? I seem to have no data output from my 6pin BME280 sensor, only showing the same stats as flagged above by TI2GBB. Everything is connected as detailed. 

I have downgraded the SparkFun BME280 library to V2.08 however I get several errors when trying to reflash the ESP8266 with this library version. 

I'm only a newby to Arduino so any assistance you can provide me in very basic language would be much appreciated.

I have flashed the ESP8266 with the V1.1 code and have the Unit talking to my WiFi etc. It also shows as connecting to the APRS Server when I undertake a test using the config menu but doesn't appear on APRS.fi? I have inputted my correct coordinates using APRSDroid but still nothing.

I am soooo frustrated now having spent over 12hrs on this project 😢

Thanks, Duncan - VK6GHZ

**Comment by tk5ep** [comment]
Hi,

I installed the 1.1f in a NodeMCU but didn't get any reading from the BME280.

I noticed that the Wire library is NOT initialized in the sketch.

I added the following line arounf line number 466 in the sketch to make it work.

  Serial.begin(SER_MON_BAUDRATE);  // to read the esp8266 core and startup messages
  Serial.println();
  delay(10);

  **Wire.begin();**

  SPIFFS.begin(); // mount SPIFFS

I hope this helps ?
Patrick
